modules/RsPaymentTerminal/manifest.yaml

```diff
@@ -0,0 +1,61 @@
+description: Payment terminal module for Feig devices.
```

Comment:
nit: defaults for `ip`, `feig_serial` and `terminal_id` (and maybe `password`) do not look like reasonable. I'd recommend to use no default and therefore enforce proper configuration

Reply:
true - was more like documentation feature -